app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
```

Comment:
`name` shouldn't also has a `presence: true` while `description` set it as false, then in the migration desciption can have default value as `""`?